src/azure-cli/azure/cli/command_modules/cdn/_params.py

```diff
@@ -501,8 +501,11 @@ def configure_log_analytic_common_parameters(c):
 # pylint: disable=protected-access
 def configure_rule_parameters(c):
     c.argument('rule_name', help='Name of the rule.')
-    c.argument('order', help='The order of the rule. The order number must start from 0 and consecutive. '
-                             "Rule with higher order will be applied later.")
+    c.argument('order', type=int,
+               help='The order in which the rules are applied for the endpoint. Possible values {0,1,2,3,………}. '
+               'A rule with a lesser order will be applied before a rule with a greater order. '
```

Comment:
less order?

Reply:
That is copied from the swagger definition, :(. 
Will change to "A rule with a lower order will be applied before one with a higher order."